src/sqsd.groovy

```diff
@@ -104,7 +105,15 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
         def resp = new RESTClient(httpHost).post(
                 path : httpPath,
                 body : payload,
-                contentType : contentType
+                contentType : contentType,
+                headers: [
```

Comment:
I guess these should be included only if they exists. I would build the map first by simply iterating `message.getMessageAttributes()`.